.github/workflows/test.yml

```diff
@@ -273,9 +270,28 @@ jobs:
     runs-on: ubuntu-latest
 
     strategy:
-      fail-fast: true
+      # Do not cancel in-progress jobs upon failure
+      fail-fast: false
+      # Single dimension matrix
       matrix:
-        machines: [1, 2, 3, 4]
+        spec:
+          [
+            "*.spec.js",
+            "AD integration/**/*.spec.js",
+            "admin/**/*.spec.js",
+            "checker/**/*.spec.js",
+            "dashboard/**/*.spec.js",
+            "feedback/**/*.spec.js",
+            "login/**/*.spec.js",
+            "maker/**/*.spec.js",
+            "number-generator/**/*.spec.js",
+            "read-only/**/*.spec.js",
+            "reports/**/*.spec.js",
+          ]
```

Comment:
Does this mean we need to remember to update this list whenever we add new tests or rename old tests? Is there any way we can make sure we've not missed any?

Reply:
Yes unfortunately, however in future this will be automated.
As long as we add under of the directories then we don't need to worry about it, only if we add a new directory.